pipes/WDL/workflows/train_16S_classifier.wdl

```diff
@@ -0,0 +1,26 @@
+version 1.0
+
+import "../tasks/tasks_16S_amplicon.wdl" as qiime
+
+workflow train_classifier_16S {
+    meta {
+        description: "User imports OTU database that will be trained on your primer sequences. All outputs can be visualized using https://view.qiime2.org/"
+        author: "Broad Viral Genomics"
+        email: "viral-ngs@broadinstitue.org"
+        allowNestedInputs: true 
+    }
+    input {
+       File     otu_ref
+       File     taxanomy_ref
+       String   forward_adapter
+       String   reverse_adapter   
+    }
+
+    call qiime.train_classifier { 
+        input: 
+            otu_ref = otu_ref,
+            taxanomy_ref = taxanomy_ref,
+            forward_adapter = forward_adapter,
+            reverse_adapter = reverse_adapter
+    }
```

Comment:
```suggestion
    call qiime.train_classifier
```

I don't think you need all that boilerplate -- just call the task with unbound inputs when you have a 1-task workflow.

Reply:
Examples of this precedent:
- https://github.com/broadinstitute/viral-pipelines/blob/master/pipes/WDL/workflows/augur_export_only.wdl
- https://github.com/broadinstitute/viral-pipelines/blob/master/pipes/WDL/workflows/downsample.wdl
- https://github.com/broadinstitute/viral-pipelines/blob/master/pipes/WDL/workflows/mafft.wdl
- https://github.com/broadinstitute/viral-pipelines/blob/master/pipes/WDL/workflows/nextclade_single.wdl